src/Illuminate/Console/Scheduling/Event.php

```diff
@@ -114,6 +114,17 @@ class Event
     public function __construct($command)
     {
         $this->command = $command;
+        $this->output = $this->getDefaultOutput();
+    }
+
+    /**
+     * Get the default output depending on the OS.
+     *
+     * @return string
+     */
+    private function getDefaultOutput()
+    {
+        return (strpos(strtoupper(PHP_OS), 'WIN')===0) ? 'NUL' : '/dev/null';
```

Comment:
I suppose you are referring to code segment and I tried to separate the commit blocks, but Git have done that automatically. Can I separate them without change the assignment order or method location?